### PR TITLE
various casks: fixes for string values

### PR DESCRIPTION
**Pull request by EricFromCanada**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.
---

### CODE REVIEW

File: Casks/h/hacker-menu.rb
Comment (krehel) [inline Casks/h/hacker-menu.rb:1]: Last release was in 2015 and upstream site no longer exists.  Probably just deprecate this imho.

File: Casks/h/hammerspoon.rb [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
     # Specific build provided for Mojave upstream https://github.com/Hammerspoon/hammerspoon/issues/3023#issuecomment-992980087
     livecheck do
-      skip "Specific build for Mojave and earlier"
+      skip "Specific build for Mojave and later"
```
Comment (bevanjkay) [inline Casks/h/hammerspoon.rb:11]: Maybe Mojave and older?

Reads to me as needed than Mojave with this change

File: Casks/g/githubpulse.rb
Comment (krehel) [inline Casks/g/githubpulse.rb:1]: This also looks abandoned.  Last update in 2015.

File: Casks/h/hammerspoon.rb (hunk 1 above)
Comment (EricFromCanada) [reply Casks/h/hammerspoon.rb:11]: According to the [link in the comment](https://github.com/Hammerspoon/hammerspoon/issues/3023#issuecomment-992980087) this build has its minimum macOS version set to 10.14.

**Review by bevanjkay — APPROVED** [review]

**Pull request merged**